config/locales/event.en.yml

```diff
@@ -66,3 +66,8 @@ en:
     capture_reason_for_lateness: Record reason for lateness
     capture_reason_for_lateness_add: Reason for lateness added
     capture_reason_for_lateness_change: Reason for lateness changed
+    mark_as_partial_case: Case marked as SSCL partial case
+    unmark_as_partial_case: Case marked as not a SSCL partial case
+    mark_as_further_actions_required: Case marked as SSCL managing case
+    unmark_as_further_actions_required: Case marked as SSCL not managing case
+    mark_as_awaiting_response_for_partial_case: Case marked as awaiting response following SSCL letter
```

Comment:
Do we need to say "Awaiting" with upper case A? Just a question as to how the states are written when in sentence.

Reply:
hmm,  I don't know it, I just made a copy of Ryan and NIcols design,  in order to save time to discuss it again,  I just leave it as it is, hopefully you don't mind 😳